index.html

```diff
@@ -1,223 +0,0 @@
-            <p class='date-text'>JULY 28</p>
-          </div>
-          <div class='other-post-text'>
-            <p class='side-header'>ONE MORE POST</p>
-            <p class='sidebar-text'>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Nulla, fuga vero? Eaque
-              possimus laboriosam veniam!</p>
-          </div>
-        </div>
-        <div class='post-link'>
-          <div class='date-button'>
-            <p class='date-text'>JULY 24</p>
-          </div>
-          <div class='other-post-text'>
-            <p class='side-header'>AND ANOTHER ONE</p>
-            <p class='sidebar-text'>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Nulla, fuga vero? Eaque
-              possimus laboriosam veniam!</p>
-          </div>
-        </div>
-      </div>
-      <div id='sidebar-notes'>
-        <div class='sidebar-pic-posts'>
-          <p class='side-header'>POLE OF NOTE</p>
-          <div class='pic-post-wrapper'>
-            <img class='pic-post-pic' src='images/dick_pole_square.png' />
-            <p class='sidebar-text'>Lorem ipsum dolor sit amet consectetur adipisicing elit. Recusandae saepe, iure
-              voluptatum ipsa excepturi omnis alias quos iste accusamus dolorum quod tempore nostrum? Ratione, expedita.
-              Nulla saepe rerum nihil porro.</p>
-          </div>
-          <div class='button-bar'>
-            <div class='learn-more-button button'>
-              <img id='btn-image' src='images/button-img.png' />
-              <p class='btn-text'>LEARN MORE</p>
-            </div>
-          </div>
-        </div>
-        <div class='sidebar-pic-posts bottom-post'>
-          <p class='side-header'>SOMETHING OF NOTE</p>
-          <div class='pic-post-wrapper'>
-            <img class='pic-post-pic' src='images/richard_ayoade_square.png' />
-            <p class='sidebar-text'>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Accusantium cupiditate
-              suscipit aspernatur illo nisi explicabo blanditiis consectetur sit minima vero. Sed nam molestias magnam
-              illum eos quaerat, velit soluta maiores?</p>
-          </div>
-          <div class='button-bar'>
-            <div class='learn-more-button button'>
-              <img id='btn-image' src='images/button-img.png' />
-              <p class='btn-text'>LEARN MORE</p>
-            </div>
-          </div>
-        </div>
-      </div>
-    </div>
-  </section>
-  <!--Footer with buttons, contact info and email input areas. JS is linked at the button to create functional buttons-->
-  <section id="footer">
-    <p class='subtitle'> QUESTIONS OR COMMENTS? <strong>GET IN TOUCH:</strong></p>
-    <div id='footer-content'>
-      <div id='footer-input-contact'>
-        <div id='input'>
-          <div id='name-email'>
-            <input id='name' class='info-input' type='text' placeholder='name' />
-            <input id='email' class='info-input' type='text' placeholder='email' />

```

Comment:
There is some controversy over whether or not to use the placeholder attribute. According to MDN, it is an [archived attribute](https://developer.mozilla.org/en-US/docs/Archive/Mozilla/XUL/Attribute/placeholder) that should not be used in [new websites](https://developer.mozilla.org/en-US/docs/Archive). Here are also a couple articles talking about this topic: [Here](https://www.smashingmagazine.com/2018/06/placeholder-attribute/) and [here](https://developer.paciellogroup.com/blog/2011/02/html5-accessibility-chops-the-placeholder-attribute/).